### PR TITLE
EWL-6562 mobile search drawer

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- [EWL-6562: Global nav - search component required for mobile](https://issues.ama-assn.org/browse/EWL-6562)

## Description:
Add mobile search drawer

## To Test:
- refer to PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/1032 for testing instructions

## Automated Test:
n/a

## Relevant Screenshots/GIFs:
screen shot 2018-11-07 at 12 41 40 pm

## Additional Notes:
There is a D8 depedency 
https://github.com/AmericanMedicalAssociation/ama-d8/pull/1032

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

File: styleguide/source/assets/js/main-navigation.js [hunk 1]
```diff
@@ -19,13 +21,15 @@
...
       $(document).click(function(e) {
-        if (!categoryNavigationMenu.is(e.target) && categoryNavigationMenu.has(e.target).length === 0) {
+        if (!$categoryNavigationMenu.is(e.target) && $categoryNavigationMenu.has(e.target).length === 0) {
           $('#global-menu').prop('checked',false);
           hideShow();
         }
       });
 
-
+      $($mobileSearchTrigger).click(function(e) {
```
Comment (nstriedinger) [inline styleguide/source/assets/js/main-navigation.js]:
```suggestion
      $($mobileSearchTrigger).unbind('click').click(function(e) {
```

File: styleguide/source/assets/js/main-navigation.js [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
     attach: function (context, settings) {
 
-      var categoryNavigationMenu = $('.ama_category_navigation_menu');
+      var $categoryNavigationMenu = $('.ama_category_navigation_menu');
+      var $mobileSearchTrigger = $('.global-search-trigger');
+      var $mobileSearch = $('.ama__global-search');
 
       // Hide/Show menu
       function hideShow() {
         if($('#global-menu').prop('checked')) {
```
Comment (nstriedinger) [inline styleguide/source/assets/js/main-navigation.js]:
```suggestion
        if ($('#global-menu').prop('checked')) {
```

File: styleguide/source/assets/js/main-navigation.js [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
-          categoryNavigationMenu.slideDown();
+          $categoryNavigationMenu.slideDown();
         } else {
```
Comment (nstriedinger) [inline styleguide/source/assets/js/main-navigation.js]:
```suggestion
        } 
        else {
```

**Review by nstriedinger — CHANGES_REQUESTED** [review]
@designerneil Take a look at this stuff

**Comment by nstriedinger** [comment]
Some oddities here.
screen shot 2018-11-07 at 4 02 33 pm

**Comment by designerneil** [comment]
@nstriedinger the button staying purple shouldn't be a problem because it's indicating the search drawer is active

**Comment by damontgomery** [comment]
@nstriedinger, If this is good to merge, can you update your review and remove the `ready for review` labels?

**Pull request merged**